stricher/week_02/FizzBuzz.py

```diff
@@ -0,0 +1,24 @@
+
+# Write a program that prints the numbers from 1 to 100 inclusive.
+# But for multiples of three print "Fizz" instead of the number
+# For the multiples of five print "Buzz".
+# For numbers which are multiples of both three and five print "FizzBuzz" instead.
+
```

Comment:
There is no call to the fizzbuzz function.  I would also suggest using print instead of ostream unless you are trying to do something specific with it that I don't know.